tests/integration/runnable/runnable_test.go

```diff
@@ -259,6 +260,55 @@ var _ = Describe("Stamping a resource on Runnable Creation", func() {
 					}))
 				})
 			})
+
+			Context("and a second Runnable matches the RunTemplateRef", func() {
```

Comment:
I'm not a fan of this style of regression test, as it's key assertion is blindly waiting for something to not happen. As such, it either has to hope it has waited long enough (else you get false positives), or always has to wait longer than necessary, protracting test execution time.

To be clear, I'm not saying this test is worthless, but my calculus was: If I know we already have tests like this for a single object only being stamped once in here (per input change) which implies the Cache, then is such an integration test much akin to a corner case test for a `Repository -> Cache` bug? If so, what value does such an integration test add that makes it worth paying the time / uncertainty cost?

Interestingly, I think if we are destined to ultimately use ReconcilerTestSuite, then this and similarly structured tests would be converted to forms that reduce the cost (increased test execution time) and eliminate uncertainty.